consent-ui/src/app/ais/entry-page/initiation/accounts/accounts-consent-review/accounts-consent-review.component.ts

```diff
@@ -69,4 +85,34 @@ export class AccountsConsentReviewComponent implements OnInit {
   onBack() {
     this.location.back();
   }
+
+  private createForm() {
+    this.targetForm = this.formBuilder.group({
```

Comment:
Please rename targetForm to consentReviewForm

Reply:
renamed